packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-breadCrumbs']"
```

Comment:
Sorry to be a pain – is it worth getting rid of the camel casin everywhere to keep it consistent with the changes made for the tests? So just updating to `breadcrumbs` or `Breadcrumbs` everywhere then.

Reply:
agreed pushed a change